spec/models/spree/image_spec.rb

```diff
@@ -55,9 +55,14 @@ module Spree
 
       context "when using public images" do
         it "returns the direct URL for the processed image" do
+          allow(ENV).to receive(:[])
```

Comment:
I was wondering if this is necessary, after trying to remove it I can confirm it is :)